packages/main/src/components/AnalyticalTable/defaults/Column/Expandable.tsx

```diff
@@ -27,16 +34,30 @@ const getPadding = (level) => {
   }
 };
 
+const useStyles = createUseStyles(
+  {
+    container: {
+      height: CssSizeVariables.sapWcrAnalyticalTableExpandButtonHeight,
+      marginInlineEnd: '2px'
```

Comment:
I followed the SAPUI5 implementation:
<img width="219" alt="image" src="https://user-images.githubusercontent.com/9749730/219285855-3078927d-02e6-438e-84c5-45a8f0ce1236.png">

But I don't have a hard opinion on that and since we're using font-size dependent values almost everywhere, I'll change it.

Reply:
If SAPUI5 does it this way I'm fine with that too - they will have their reasons :)